plugin/src/main/kotlin/com/varabyte/kobweb/intellij/project/KobwebProjectResolver.kt

```diff
@@ -0,0 +1,49 @@
+package com.varabyte.kobweb.intellij.project
+
+import com.intellij.openapi.externalSystem.model.DataNode
+import com.intellij.openapi.externalSystem.model.Key
+import com.intellij.openapi.externalSystem.model.project.ModuleData
+import com.intellij.openapi.externalSystem.util.ExternalSystemApiUtil
+import com.intellij.openapi.module.Module
+import com.varabyte.kobweb.intellij.model.KobwebModel
+import com.varabyte.kobweb.intellij.model.gradle.tooling.KobwebModelBuilderService
+import org.gradle.tooling.model.idea.IdeaModule
+import org.jetbrains.plugins.gradle.service.project.AbstractProjectResolverExtension
+import org.jetbrains.plugins.gradle.util.GradleConstants
+
+/**
+ * A project resolver that extends a module with information about its Kobweb contents (if any).
+ */
+class KobwebProjectResolver : AbstractProjectResolverExtension() {
```

Comment:
Nit: `KobwebGradleProjectResolver` would probably be a better name

Reply:
Agreed, done.